slurk/views/login/__init__.py

```diff
@@ -15,7 +15,7 @@
 from slurk.models import User, Token
 
 from .forms import LoginForm
-from .events import *  # NOQA
+import slurk.views.login.events  # NOQA
```

Comment:
Do we still need `# NOQA`?

Reply:
flake8 says 'yes' ... I assume the reason why you initally removed this import `'slurk.views.login.events' imported but unused`